### PR TITLE
Add `quiet` kwarg to `Packages.load`

### PR DESCRIPTION
**Pull request by fingolfin**
This is useful for debugging among other things.

Also reformat some docstrings so that they render nicer in the HTML
manual (and don't require the user to scroll horizontally to read them)

### CODE REVIEW

File: src/packages.jl [hunk 1]
```diff
@@ -32,19 +32,22 @@ the package banner is not printed.
 If `install` is set to `true` and the required GAP package is not yet
 installed then [`install`](@ref) is called first, in order to install
 the newest released version of the package.
+
+If `quiet` is set to `true` then package banners are shown for all packages
```
Comment (ThomasBreuer) [inline src/packages.jl:36]: "is set to `false`" ?

File: src/packages.jl [hunk 2]
```diff
@@ -32,19 +32,22 @@ the package banner is not printed.
...
     gspec = GapObj(spec)
     gversion = GapObj(version)
-    loaded = Globals.LoadPackage(gspec, gversion, false)
+    loaded = Globals.LoadPackage(gspec, gversion, !quiet)
     if loaded == true
         return true
     elseif install == true
         # Try to install the package, without showing messages.
-        if Packages.install(spec; interactive = false, quiet = true)
+        if Packages.install(spec; interactive = false, quiet)
```
Comment (ThomasBreuer) [inline src/packages.jl:48]: `Packages.install(spec; interactive = false, quiet = quiet)`?

**Review by ThomasBreuer — COMMENTED** [review]
just two small comments

File: src/packages.jl (hunk 1 above)
Comment (fingolfin) [reply src/packages.jl:36]: Indeed.

File: src/packages.jl (hunk 2 above)
Comment (fingolfin) [reply src/packages.jl:48]: That's equivalent; Julia 1.6 allows replacing kwargs of the form `foo = foo` by just `foo`

Comment (ThomasBreuer) [reply src/packages.jl:48]: O.k.
(I do not like this syntax. The left hand side of `foo = foo` is a key, the right hand side is a variable.
When one copies such a code line into another context and adjusts the variable name then one gets into trouble.)

**Pull request merged**